CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -0,0 +1,85 @@
+# Changelog 1.4
+
+## 1.4.0 - 2021-11-03
+
+The NodePortLocal feature is graduated from Alpha to Beta.
+
+### Added
+
+- Support for proxying all Service traffic, including NodePort, LoadBalancer, and ClusterIP traffic. Therefore, running kube-proxy is no longer required. ([#2599](https://github.com/antrea-io/antrea/pull/2599) [#2235](https://github.com/antrea-io/antrea/pull/2235) [#2897](https://github.com/antrea-io/antrea/pull/2897) [#2863](https://github.com/antrea-io/antrea/pull/2863), [@hongliangl] [@lzhecheng])
+  * The feature works for both Linux and Windows
+  * The feature is experimental and therefore disabled by default. Use the `antreaProxy.proxyAll` configuration parameter for the Antrea Agent to enable it
+  * If kube-proxy is removed, the `kubeAPIServerOverride` configuration parameter for the Antrea Agent must be set to access apiserver directly.
+- Add [AntreaIPAM] feature to allow flexible control over Pod IP Addressing. ([#2956](https://github.com/antrea-io/antrea/pull/2956), [@gran-vmv] [@annakhm])
+  * Add new IPPool API to define ranges of IP addresses which can be used as Pod IPs
+  * A Pod's IP will be allocated from the IPPool specified by the `ipam.antrea.io/ippools` annotation of the Pod's Namespace if there is one
```

Comment:
We should mention data path changes to forward traffic of AntreaIPAM Pods through VLAN network.

Reply:
https://github.com/antrea-io/antrea/blob/main/docs/feature-gates.md?plain=1#L224
This is the requirements.
```
In Antrea 1.4, AntreaIPAM can only work with Linux+IPv4, system OVSDatapathType and
noEncap, noSNAT mode.
AntreaIPAM is only verified with other FeatureGates at default state.
```

Data path change:
Enable AntreaIPAM will bridge Node uplink to OVS switch, which called "bridging mode". And will be restored when antrea-agent shutdown gracefully. 
Node may lose network connection when antrea-agent stopped unexpectedly, and users can reboot the Node to recover.
AntreaIPAM Pods will not be routed by Node network stack.
All intra-Node traffic's destination MAC will be overwritten to correct one and will be sent to destination OVS port directly, although AntreaIPAM Pods and Per-Node IPAM Pods are not in same subnet.

And we also need to add AntreaIPAM to the list:
https://github.com/antrea-io/antrea/blob/main/docs/feature-gates.md?plain=1#L46
`| AntreaIPAM              | Agent + Controller | false | Alpha | v1.4          | N/A          | N/A        | Yes                |       |`

Reply:
Can you create a PR for the required documentation change?

Reply:
Sure.

Reply:
https://github.com/antrea-io/antrea/pull/2971